api/src/main/java/com/epam/pipeline/entity/metadata/MetadataField.java

```diff
@@ -17,21 +17,28 @@
 package com.epam.pipeline.entity.metadata;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
-import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Getter
 @Setter
 @NoArgsConstructor
-@AllArgsConstructor
 public class MetadataField {
     private String name;
     @JsonIgnore
     private String dbName;
     private boolean predefined = false;
 
+    //todo: Replace with @AllArgsConstructor once lombok version 1.16.20 is used.
+    // The current version causes JsonMappingException.
+    // See more https://stackoverflow.com/questions/40546508/jsoncreator-could-not-find-creator-property-with-name-even-with-ignoreunknown
```

Comment:
Could you please suppress checkstyle warning for this line using `SuppressWarnings` annotation?